### ISSUE TITLE
Add indicator that there is an active search

### ISSUE THREAD

**Issue by alSeveR** [nextcloud/server#36158]
In many cases users perform a search and do not delete it. It is possible to change directory and have the feeling that something has been deleted, as there is an invisible active filter.

I think the icon could be changed when there is an active search. Something similar to the notification icon when there are unread notifications.

Another alternative would be that the search box cannot be hidden while the search is active.

Thanks in advance for considering this.

**Comment by vdtoorn** [comment]
I agree with the problem, but a different solution may be to show the search text if it is still active

**Comment by susnux** [comment]
With Nextcloud 30 a chip is shown on the top of the files list for every active filter. Including search.
